orchestration/src/main/java/com/sap/ai/sdk/orchestration/AssistantMessage.java

```diff
@@ -0,0 +1,27 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import javax.annotation.Nonnull;
+import lombok.Value;
+import lombok.experimental.Accessors;
+
+/** Represents a chat message as 'assistant' to the orchestration service. */
+@Value
+@Accessors(fluent = true)
+public class AssistantMessage implements Message {
+
+  /** The role of the assistant. */
+  @Nonnull public static final String ROLE = "assistant";
+
+  @Nonnull String content;
+
+  /**
+   * Converts the message to a serializable ChatMessage object.
+   *
+   * @return the corresponding {@code ChatMessage} object.
+   */
+  @Nonnull
+  public ChatMessage toChatMessage() {
```

Comment:
(Question/Minor)

Why is this public?

Reply:
The practice of interface with public method is already established with ContentFilter and MaskingProvider

Reply:
Useful when the user creates a model.Template (which is generated) with the help of convenience API.